### PR TITLE
[odla_dnnl][bugfix] olda_prelu use wrong dims to multiply

### PR DESCRIPTION
**Pull request by xuhongyao**
if the shape of input is {1, c, h, w} & slope is {c} in odla_PRelu,
it will occur "assert(dims_rhs.dims[j] == 1)" in odla_dnnl.cc line 550

### CODE REVIEW

File: ODLA/platforms/dnnl/odla_dnnl.cc [hunk 1]
```diff
@@ -644,8 +644,26 @@ odla_value odla_PRelu(odla_value input, odla_value slope,
...
+    for (int i = 0; i < dims_input.size; i++) {
+      if (dims_input.dims[i] == dims_slope.dims[0]) {
+        slope_shape.dims[i] = dims_slope.dims[0];
+      }
+      slope_shape.dims[i] = 1;
+    }
+    new_slope = CreateValue(slope->mem, slope_shape, value_id);
+  }
+
+  auto neg_slop = unary_eltwise_op(dnnl::algorithm::eltwise_linear, new_slope,
```
Comment (weimingzha0) [inline ODLA/platforms/dnnl/odla_dnnl.cc:665]: It looks like "unary_eltwise_op" should support broadcasting rule, doesn't it?
PRelu itself is insensitive to NHWC/NCHW.

**Comment by ahuizxc** [comment]
thanks, i think it is better move this function to binary_eltwise() as it can fix all this kind of error.

File: ODLA/platforms/dnnl/odla_dnnl.cc (hunk 1 above)
Comment (xuhongyao) [reply ODLA/platforms/dnnl/odla_dnnl.cc:665]: yes,  if all devices have a limit that input and slope have same dims, we even can  implement it in halo

Comment (ahuizxc) [reply ODLA/platforms/dnnl/odla_dnnl.cc:665]: hi hongyao, as matter of fact,  we should both check lhs and rhs value when call `binary_eltwise()` so I fix it once for all :) see #116

**Pull request closed without merging**